x-pack/plugins/secops/public/components/timeline/timeline.test.tsx

```diff
@@ -173,7 +173,7 @@ describe('Timeline', () => {
         );
 
         wrapper
-          .find('[data-test-subj="rangePicker"]')
+          .find('[data-test-subj="rangePicker"] select')
```

Comment:
Note: This was a weird one, but I had to use a more specific selector to get to the `select` element in order for the tests to work with `EuiSelect`